.github/workflows/aws-deploy.yaml

```diff
@@ -12,70 +12,26 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: pre-commit/action@v2.0.0
-  sceptre-sandbox:
-    if: github.event_name == 'push'
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-deploy:
+    name: Deploy CloudFormation templates using sceptre
     runs-on: ubuntu-latest
     needs: pre-commit
+    if: github.ref == 'refs/heads/prod'
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.8
```

Comment:
why you remove this?  don't you need to setup python?

Reply:
This setup is based off of what we did with for the [Nextflow accounts](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/blob/prod/.github/workflows/aws-deploy.yaml). Sceptre and its dependencies are included in the container used by the sceptre-action.

Reply:
ahh yes of course.